content/en/docs/tasks/administer-cluster/migrating-to-cloud-controller-manager.md

```diff
@@ -0,0 +1,175 @@
+---
+reviewers:
+- andrewsykim
+title: Migrating to Cloud Controller Manager
+content_template: templates/task
+min-kubernetes-server-version: v1.16
+---
+
+{{% capture overview %}}
+
+Cloud providers used to be integrated into the Kubernetes Controller Manager (KCM).
+In Kubernetes 1.17, this changes, and cloud platform integrations must run in a
+dedicated Cloud Controller Manager (CCM) that is specific to each cloud platform.
+
+This document describes how to migrate from the integrated cloud provider in KCM
+to a dedicated CCM.
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+You need to have a Kubernetes cluster, and the kubectl command-line tool must
+be configured to communicate with your cluster. {{< version-check >}}
+
+It's also assumed that you are currently running one of the cloud integrations
+provided by `kube-controller-manager`. This daemon may be running either as
+a pod or as a standalone service.
+
+### As a Pod
+
+Find out which cloud provider you are using by running the following command:
+
+```shell
+kubectl describe pod kube-controller-manager -n kube-system
+```
+
+### As a systemd service
+
+Find the `kube-controller-manager` parameters with the following command:
+
+```shell
+systemctl status | grep kube-controller-manager
+```
+
+Look for the `--cloud-provider` option, which will show you which cloud
+provider is used.
+
+If the option says `--cloud-provider=external`, you are already running an
+external cloud provider. If the option is not set at all, you're most likely not
+using cloud integration. In this case, you don't need to migrate.
+
+Once you've determined which cloud provide you're using, ensure that an external
+alternative is available. Note that there may be several alternatives, with
+different features. Choose a provider that has roughly the same feature set or,
+even better, is based on the former integrated provider.
+
+**TODO** Links to external providers here
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Preparation
+
+### Determine compatibility issues between internal and external cloud provider
+
+* Missing features
```

Comment:
I think this is too broad to be useful to users. Wondering if we should pick one provider and walk through it in detail. What do you think?